contrib/to-a11y-string/test/to-a11y-string-spec.js

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable max-len */
+// @flow
+import {renderString} from "../to-a11y-string";
+
+describe("renderString", () => {
+    describe("basic expressions", () => {
+        test("simple addition", () => {
+            const result = renderString("1 + 2");
+            expect(result).toMatchInlineSnapshot(`"1, plus, 2"`);
+        });
+    });
+
+    describe("accent", () => {
+        test("\\vec", () => {
+            const result = renderString("\\vec{a}");
+            expect(result).toMatchInlineSnapshot(`"a, with, vector, on top"`);
+        });
+
+        test("\\hat{a}", () => {
+            const result = renderString("\\hat{a}");
+            expect(result).toMatchInlineSnapshot(`"a, with, hat, on top"`);
+        });
+    });
+
+    describe("accentUnder", () => {
+        test("\\underleftarrow", () => {
+            const result = renderString("\\underleftarrow{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"1, plus, 2, with, left arrow, underneath"`,
+            );
+        });
+
+        test("\\underlinesegment", () => {
+            const result = renderString("\\underlinesegment{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"1, plus, 2, with, line segment, underneath"`,
+            );
+        });
+    });
+
+    describe("color", () => {
+        test("\\color{red}", () => {
+            const result = renderString("\\color{red}1+2");
+            expect(result).toMatchInlineSnapshot(
+                `"start color red, 1, end color red, plus, 2"`,
+            );
+        });
```

Comment:
That looks like old `\color` behavior. It won't work without option `colorIsTextColor` set to `true`.

Reply:
That's for pointing that out.  I'll fix this.